fleet_management_api/controllers/tenant_controller.py

```
@@ -8,6 +8,21 @@
+def create_tenants(tenant):  # noqa: E501
+    """Create new Tenants.
+
+     # noqa: E501
+
+    :param tenant: Tenants to be created.
+    :type tenant: list | bytes
+
+    :rtype: Union[None, Tuple[None, int], Tuple[None, int, Dict[str, str]]
+    """
+    if connexion.request.is_json:
+        tenant = [Tenant.from_dict(d) for d in connexion.request.get_json()]  # noqa: E501
```

Ruff (0.8.2)

Local variable `tenant` is assigned to but never used

Remove assignment to unused variable `tenant`

(F841)